rules/windows/execution_unusual_dns_service_children.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = "An unexpected process spawned from dns.exe, which is the process responsible for DNS server services."
```

This doesn't actually explain the rule itself which it should.
I think the title should capture what this rule is trying to do. But it's written from the perspective of a description of an alert itself. Too much of the info is in `note`, and could be pulled up to `description`

---

```suggestion
description = "Unusual child process of dns.exe.  This rule alerts on all processes spawning from dns.exe that are not conhost.exe.  Adversaries might use a variety of mechanisms to induce this behavior and gain code execution, one of which is exploitation of the dns.exe process."
```

---

I don't think we should repeat the `name` of a rule in the description, but I like what you have, how about:

```suggestion
description = "An unexpected process spawned from dns.exe, which is the process responsible for DNS server services. Adversaries might use a variety of mechanisms to exploit this service to achieve code execution.""
```

---

```suggestion
description = "Identifies an unexpected process spawning from dns.exe, which is responsible for DNS server services. Adversaries might use a variety of mechanisms to exploit this service to achieve code execution."
```

I usually tend to go with Identifies... X

---

sounds good. just change the tense to describe the rule instead of an individual alert.